src/Control/Effect/Class.hs

```diff
@@ -16,73 +16,87 @@ import GHC.Generics
 -- | The class of effect types, which must:
 --
 --   1. Be functorial in their last two arguments, and
---   2. Support threading effects in higher-order positions through using the carrier’s suspended state.
+--   2. Support threading effects in higher-order positions through using the carrier’s suspended context.
```

Comment:
As discussed [in #296](https://github.com/fused-effects/fused-effects/pull/296/files#r339115293), I’m now calling this parameter the context, or context functor. Unlike #296, I’ve also gone through and changed the names of all the parameters in all the `Effect` instances.